lib/recurly/client.php

```diff
@@ -162,6 +178,13 @@ private function _sendRequest($method, $uri, $data = '')
     }
     $headers = $this->_getHeaders($header);
 
+    if (Recurly_Client::$logger) {
+      Recurly_Client::$logger->debug('Response from Recurly', array('headers' => $headers));
+      if ($body) {
+        Recurly_Client::$logger->debug('Recurly response body', array('body' => $body));
+      }
+    }
```

Comment:
And here playing with:
```
    if (Recurly_Client::$logger) {
      Recurly_Client::$logger->info('Recurly Response', array('code' => $statusCode, 'method' => $method, 'uri' => $uri));
      Recurly_Client::$logger->debug('Response data', array('headers' => $headers, 'body' => $body));
    }
```
I think including the status code would be helpful... though I'm realizing we need to add a logging call ahead of the `      $this->_raiseCurlError()` because it'll throw and skip this.